backend/shared/precommit-setup.cfg

```diff
@@ -0,0 +1,32 @@
+[pep8]
+max-line-length = 120
+exclude = *migrations*
+ignore = E309
+
+[flake8]
+exclude = migrations
+max-line-length = 120
+max-complexity = 10
+
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = kesaseteli.settings
```

Comment:
Good idea to share the config.
The file is in shared folded but references kesaseteli.settings. How to work around this?

Reply:
I think the `tool:pytest` section can be removed from the common setup.cfg for now since it is not used in the pre-commit hook. I guess the best option is to still have the individual `setup.cfg` in each folder as well.